mantidproject/mantid
# Fix positions of the 4 SEQ new det packs at row A

**Description of work.**
In current SEQ instrument def file https://github.com/mantidproject/mantid/blob/de4e3123bf0397fb3e3821c5d868ffc6e833061c/instrument/SEQUOIA_Definition.xml#L58
the packs were placed to align with row B. Should be aligned to row C instead. It causes the incident energy calculation to be off for those packs. This PR updated their positions using corresponding packs in row C. More careful calibration will be done later.

Before this fix
![Screen Shot 2019-07-18 at 10 41 36 AM](https://user-images.githubusercontent.com/1796155/61467303-6264f100-a949-11e9-940a-ec853af0d4d5.png)

After this fix
![Screen Shot 2019-07-18 at 10 45 48 AM](https://user-images.githubusercontent.com/1796155/61467330-6bee5900-a949-11e9-9c8c-4e206af890d0.png)


**To test:**

* In mantid, load a recent run, for example 182230.
* run LoadInstrument and load the new xml
* run DGSReduction
* Use slice viewer to see the reduced spectrum (example above)

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
